content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -17,6 +17,15 @@ You need [`read and write variables` permissions](/docs/cloud/users-teams-organi
 - The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
 - The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
 
+## Run-Specific Variables
+
+When using Terraform 1.1 or later, you can set input variable values for a particular plan/apply run on the command line using either of the following mechanisms:
+
+* Specifying `-var` and `-var-file` arguments
+* Local environment variables named `TF_VAR_` followed by the name of a declared variable
```

Comment:
```suggestion
* Creating local environment variables prefixed with `TF_VAR`. For example, if you declare a variable called `replicas` in your configuration, you could create a local environment variable called `TF_VAR_REPLICAS` and set it to a particular value. When you use the [CLI Workflow](/docs/cloud/run/cli.html), Terraform automatically identifies these environment variables and applies their values to the run.
```

Reply:
Actually, if the variable name is "replicas", the local environment variable has to be TF_VAR_replicas

traditionally, we use lowercase identifiers for terraform variables and CAPS identifiers for environment variables.

Reply:
but you _can_ use CAPS, the terraform variables are just case-sensitive.

Reply:
I think I'll change the precedence example/illustration to lowercase to match this and reinforce the convention

Reply:
okay!! just be sure to make the image 2048 pixels wide :-)